index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Apple Watch Series 7</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+      
+      <nav id="nav-bar">
+        <div class="logo">Apple</div>
+        <ul>
+          <li class="nav-link"><a href="#About">About</a></li>
+          <li class="nav-link"><a href="#How-it-works">How it works</a></li>
+          <li class="nav-link"><a href="3contact">Contact</a></li>
+        </ul>
+      </nav>
+      <h1>The future of tech is here</h1>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nihil, aliquid.</p>
+      <img id="header-img" src="img/AS7.jpg" alt="Apple-Series-7">
+    </header>
+    <section id="About">
+      <p class="about">About The Products</p>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Facere libero cum tempora molestias recusandae at placeat accusantium, culpa quam, tempore odio enim sequi explicabo, ipsum nostrum voluptates hic dolore. Enim?</p>
+      <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/MMdQ-gWBNZE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Kindly use CSS instead of the “frameborder” attribute on the “iframe” element is no longer used.

Reply:
- [ ] Please make sure comments are clear and easy to understand.